packages/twenty-server/src/engine/workspace-manager/workspace-migration-v2/interceptors/utils/from-workspace-migration-builder-exception-to-metadata-validation-response-error.util.ts

```diff
@@ -28,6 +28,8 @@ export const fromWorkspaceMigrationBuilderExceptionToMetadataValidationResponseE
         invalidRouteTrigger: 0,
         invalidFieldMetadata: 0,
         invalidViewGroup: 0,
+        invalidRole: 0,
```

Comment:
It's a pain to always update snapshots on this, should avoid adding them if 0 same for errors array entries in response error body